addons/website/static/src/interactions/carousel_slider.js

```diff
@@ -0,0 +1,48 @@
+import { registry } from "@web/core/registry";
+import { Interaction } from "@website/core/interaction";
+import { debounced } from "@website/core/utils";
+// import { debounce } from "@web/core/utils/timing";
+
+export class CarouselSlider extends Interaction {
+    static selector = ".carousel";
+    dynamicContent = {
+        "img": {
+            "t-on-load": this.computeMaxHeight,
+        },
+        "_window": {
+            "t-on-resize": this.debounced(this.computeMaxHeight, 250),
+        },
+        ".carousel-item": {
+            "t-att-style": () => `min-height: ${this.maxHeight};`,
+        },
+    };
+    carouselOptions = undefined;
+
+    setup() {
+        this.maxHeight = undefined;
+    }
+
+    start() {
+        this.computeMaxHeight();
+        const carouselBS = window.Carousel.getOrCreateInstance(this.el, this.carouselOptions);
+        this.registerCleanup(() => carouselBS.dispose());
+    }
+
+    computeMaxHeight() {
+        this.maxHeight = undefined;
+        for (const itemEl of this.el.querySelectorAll(".carousel-item")) {
+            const isActive = itemEl.classList.contains("active");
+            itemEl.classList.add("active");
+            const rect = itemEl.getBoundingClientRect();
+            const height = rect.height;
+            if (height > this.maxHeight || this.maxHeight === undefined) {
+                this.maxHeight = height;
+            }
+            itemEl.classList.toggle("active", isActive);
+        }
+    }
```

Comment:
also, if it is debounced, you probably need to manually call this.updateContent here to make sure it is actually applied to the dom. otherwise, colibri will call it initially, but not after the debounced code is executed